src/Altinn.Platform/Altinn.Platform.Register/Altinn.Platform.Register.Tests/TestingControllers/PartiesControllerTests.cs

```diff
@@ -169,14 +187,24 @@ public async Task PostPartyLookup_InputIsOrgNo_BackendServiceRespondsWithParty_C
 
             HttpClient client = GetTestClient(partiesService.Object);
             client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+          
 
             PartyLookup lookUp = new PartyLookup { OrgNo = OrgNo }; 
 
             StringContent requestBody = new StringContent(JsonSerializer.Serialize(lookUp), Encoding.UTF8, "application/json");
 
+            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Post, "/register/api/v1/parties/lookup");
+            httpRequestMessage.Content = requestBody;
+            httpRequestMessage.Headers.Add("PlatformAccessToken", token);
 
             // Act
-            HttpResponseMessage response = await client.PostAsync("/register/api/v1/parties/lookup", requestBody);
+            HttpResponseMessage response = await client.SendAsync(httpRequestMessage);
+            // HttpResponseMessage response = await client.PostAsync("/register/api/v1/parties/lookup", requestBody);
+
+            if(!response.StatusCode.Equals(HttpStatusCode.OK))
+            {
+               string contentString = await response.Content.ReadAsStringAsync();
```

Comment:
This test now read the response.Content twice. Not sure about the intention behind this first one. The resulting string isn't used in any assert.

Reply:
This is just a trick I use when the tests fails and i need to get the stack fro localtest